src/book/03-guides/04-general/02-troubleshooting.mdx

```diff
@@ -6,7 +6,7 @@ description: Solutions to common problems you might encounter at NUbots.
 slug: /guides/general/troubleshooting
 ---
 
-# General
```

Comment:
I thought skipping header levels was naughty?

Reply:
Not the first level. The first level is whatever the title of the page is, eg Troubleshooting. Therefore we should start all headers from level 2.